public/styles/room.css

```diff
@@ -1061,13 +1076,70 @@ kbd {
 
 #searchWrapper {
   position: relative;
+  display: flex;
 }
 
 #searchBar {
   width: 100%;
-  height: 2.5rem;
-  border-radius: 3px;
+  height: 64px;
   border: 1px solid #eaeaea;
-  padding: 5px 10px;
   font-size: var(--font-size-md);
+  background-color: rgba(255, 255, 255, 0.2);
+  padding: 2px 23px 2px 46px;
+  border: 2px solid rgba(255, 255, 255, 0.2);
+  box-sizing: border-box;
+  border-radius: 12px;
+  margin-left: 2.5rem;
+  margin-right: 2.5rem;
 }
+
+::placeholder {
+  color: #D8D8D8;
+}
+
+#searchBar:hover, #searchBar:focus {
+  border: 1px solid #F7FF57;
+  background-color: #F2F2F2;
+  outline: none !important;
+} 
+
+.search-icon {
+  position: absolute;
+  top: 18px;
+  left: 55px;
+  width: 24px;
+  height: 24px;
+}
+
+.clear-icon {
+  position: absolute;
+  top: 18px;
+  right: 50px;
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+  visibility: hidden;
+}
+
+#choose-song {
```

Comment:
Generally you probably want to keep all styles isolated to class names since it would require either an id lower in the cascade or inline styles to override these styles.

Reply:
updated to:

```
.modal-song {
  ...
}
```